dotCMS/src/main/java/com/dotcms/security/apps/AppSchema.java

```diff
@@ -25,7 +26,7 @@
  *  required: false
  *
  */
-public class AppSchema {
+public class AppSchema implements Serializable{
```

Comment:
Codacy found an issue: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6315945)